test/e2e/create_test.go

```diff
@@ -143,6 +143,29 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 	require.NoError(t, err, "failed to create Windows MachineSet")
 	machines, err := tc.waitForWindowsMachines(int(gc.numberOfBYOHNodes), "Provisioned", false)
 	require.NoError(t, err, "Machines did not reach expected state")
+
+	// Change the default shell to Powershell for the first BYOH VM. We expect there to be BYOH VMs with either
+	// cmd or powershell as the default shell, so getting a mix of both, either between different BYOH VMs, or with the
+	// VMs spun up for the Machine testing is important.
+
+	// This needs to be retried as it will only succeed once the VM is fully initialized
+	// This should not increase the overall test time, as WMCO would have to wait for the VM to be ready anyway.
+	err = wait.Poll(nodeRetryInterval, time.Minute*10, func() (done bool, err error) {
+		command := "New-ItemProperty -Path \"HKLM:\\SOFTWARE\\OpenSSH\" -Name DefaultShell " +
+			"-Value \"C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe\" -PropertyType String -Force"
+		addr, err := controllers.GetAddress(machines.Items[0].Status.Addresses)
+		if err != nil {
+			log.Printf("Machine %s does not have a valid address, retrying...", machines.Items[0].GetName())
+			return false, nil
+		}
+		_, err = tc.runPowershellSSHJob("change-default-shell", command, addr)
+		if err != nil {
+			log.Printf("failed to change %s default shell to Powershell, retrying...", machines.Items[0].GetName())
+			return false, nil
+		}
+		return true, nil
+	})
```

Comment:
Please make this a separate helper method like `changeDefualtShell` or something to that effect.